### PR TITLE
Tooltip trigger

### PR DESCRIPTION
**Pull request by ddereszewski**
This Allows to trigger the tooltip either by onPress or onLongPress

### CODE REVIEW

File: src/tooltip/Tooltip.js [hunk 1]
```diff
@@ -158,9 +166,7 @@ class Tooltip extends React.PureComponent {
         ) => {
           this.setState({
             xOffset: pageOffsetX,
-            yOffset: isIOS
-              ? pageOffsetY
-              : pageOffsetY - StatusBar.currentHeight,
+            yOffset: pageOffsetY,
```
Comment (iRoachie) [inline src/tooltip/Tooltip.js]: Why'd you change this here?

**Comment by Slapbox** [comment]
@iRoachie, thanks for reviewing this!

If the author doesn't use Typescript it might not be obvious what action is needed. I also need this feature (`onLongPress`) and I'd help get this PR completed, but I don't use Typescript so I'm not sure exactly what's needed.

Are all the Typescript definitions in this file, and do we just need to add `onPress` and `onLongPress` as `void`s? https://github.com/react-native-training/react-native-elements/blob/master/src/index.d.ts#L1389-L1455

**Comment by iRoachie** [comment]
The typescript definitions mirror how the actual code would work. In this case you need to add a trigger property which type is either onPress | onLongPress

**Comment by flyingcircle** [comment]
Closing in preference of #2406

**Pull request closed without merging**